package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "client",
-  "version": "2.0.0-beta.221",
+  "version": "2.0.0-beta.219",
```

Comment:
@keyy123 why you have to change the version yourself, oh, the version is wrong? we generate it automatically, I feel you operations on git is a little bit problematic, let me find a way to help you out

Reply:
I got it, it is this commit 143665f58c744bafc6c102259f21cf61b6ffd733 cause the problem, I think we can rollback this commit and try again, did you try remove the packag-lock.json, and install npm and generate it again?

Reply:
yes

Reply:
I can try `git revert 143665f` to undo the commit history and rollback the repo to the moment before the issues happened. I think... 😅